VDA-XP-Reward/.env.example

```diff
@@ -0,0 +1,17 @@
+# Used to verify contract after deployment
+ETHERSCAN_API_KEY="<Input your etherscan.com api key>"
+POLYGONSCAN_API_KEY="<Input your polygonscan.com api key"
+
+# Used to deploy contract
+PRIVATE_KEY="<Input your wallet private key for deploying contract>"
+
+# Used for contract deployment & verification. Optional
+POLYGON_TESTNET_RPC="<Input your own RPC url for polygon mainet>"
```

Comment:
Is this TESTNET or MAINNET?

Reply:
In fact, this is the RPC of `Mumbai` testnet.
Removed.